test/testutils.js

```diff
@@ -28,7 +28,19 @@ get_server_config =
 function(config, host) {
   config = config || init_config();
   host = host || config.server_default;
-  return extend({}, config.default_server_config, config.servers[host]);
+  // Override the config with the command line if provided.
+  local = {};
+  if (process.env["npm_config_noserver"]) {
+    local["no_server"] = true;
+  } else {
+    for (var i = 0; i < process.argv.length; ++i) {
```

Comment:
Sloppy copy/pasting there. Fixed, and moved the block that was copy/pasted from into this function, too.